vtex/ab-tester
# Feat/save abtests results

#### What is the purpose of this pull request?
Save the results of all AB tests done.

#### What problem is this solving?
Currently the data is erased when the test is finished and we have no way of keeping track of tests performed, results etc.

#### How should this be manually tested?
Link the ab tester;
Start an ab test;
Finish the test;
Check if the data was saved.

#### Types of changes

A couple of things:
- I'm not sure if what I did was good practice (see comment in the code);
- I have no idea how to create automated tests for this. For the ab tester overall, actually. But will look into it soon.
